cmd/olympus/actions/actions_test.go

```diff
@@ -22,5 +22,4 @@ func (as *ActionSuite) Test_Cache_Miss_Route() {
 	mod.Version = "1.0.0"
 	res := as.JSON("/cachemiss").Post(mod)
 	as.Equal(200, res.Code)
-	as.Contains(res.Body.String(), "\"module\":\"moduleName\",\"version\":\"1.0.0\"")
```

Comment:
was a test failing because of this?

Reply:
yes, i'm not returning module info as method changes to just queue a job